src/routes/game/components/dialogs/components/RulesDialog.vue

```diff
@@ -1,217 +1,50 @@
 <template>
   <BaseDialog id="rulesDialog" max-width="1250">
```

Comment:
Let's add 
```suggestion
  <BaseDialog id="rulesDialog" max-width="1250" :persistent="false">
```

So that this closes when clicking outside of it